ArduCopter/GCS_Mavlink.cpp

```diff
@@ -991,30 +991,30 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
 
 MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_command_int_t &packet)
 {
-#if MODE_AUTO_ENABLED
-    if (copter.flightmode->mode_number() != Mode::Number::AUTO) {
-        // only supported in AUTO mode
+#if defined(MODE_AUTO_ENABLED) && defined(MODE_GUIDED_ENABLED)
```

Comment:
I am not sure whether
`#if defined(MODE_GUIDED_ENABLED) && defined(MODE_AUTO_ENABLED)
`
or
`#if defined(MODE_GUIDED_ENABLED) || defined(MODE_AUTO_ENABLED)`